VideoLocker/src/main/java/org/edx/mobile/view/CourseUnitWebviewFragment.java

```diff
@@ -71,6 +81,10 @@ public void onDestroy(){
         super.onDestroy();
     }
 
+    public void onEvent(NetworkConnectivityChangeEvent event){
+        setFragmentState(!NetworkUtil.isConnected(getContext()));
```

Comment:
We should either make sure that we use `registerSticky()` to receive the last posted status during initialization (and make sure that the initial state is always posted during app initialization), or we should call this explicitly from `onViewCreated()` or some other callback for Fragment initialization.